docs/01-startgids.md

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
* Dit is een Markdown file, waarom maak je gebruik van HTML?
* Er zijn erg veel anchors wat geen anchor is (geen `href`)

Reply:
En realiseer je dat er nu geen `index.md` meer is, Fractal een eigen landingspagina gebruikt? Wil je dat de bezoeker direct deze informatie ziet bij het openen van de handleiding, dan moet de filename `01-index.md` zijn

Reply:
dan verander ik de naam terug naar 01-index.md

Reply:
Ik heb alles verplaatst naar modals.md